src/service_layer/utils.py

```diff
@@ -255,3 +236,39 @@ def prepare_poll_data(api_polls):
         for api_poll in api_polls
     ]
     return polls
+
+
+def prepare_politician_data(api_politicians):
```

Comment:
The data format needs to be corrected.
I guess that you need to remove a pair of curly brackets.

Reply:
Good spotting! I removed it and it works locally